packages/ui/src/council/hooks/useCouncilRemainingPeriod.ts

```diff
@@ -34,3 +34,34 @@ export const useCouncilRemainingPeriod = (until: 'stageEnd' | 'electionEnd' = 's
     )
   }, [api?.isConnected])
 }
+
+export const useCouncilPeriodInformation = (until: 'stageEnd' | 'electionEnd' = 'stageEnd') => {
+  const { api } = useApi()
+
+  return useObservable(() => {
+    if (!api) return
+
+    const periodEnd = electionStageObservable(api).pipe(
+      map(({ stage, changedAt: start }) => {
+        const durations = [
+          api.consts.council.announcingPeriodDuration,
+          api.consts.referendum.voteStageDuration,
+          api.consts.referendum.revealStageDuration,
+          api.consts.council.idlePeriodDuration,
+        ].map(Number)
+        const position = ['announcing', 'voting', 'revealing', 'inactive'].indexOf(stage)
+        const offset = until === 'stageEnd' ? 1 : durations.length
+
+        return start + sum(durations.slice(position, position + offset))
```

Comment:
Hello @thesan  I have tried to change useCouncilRemainingPeriods function, but there are many references for this (from 5 places), so if I updated the returned data stucture, i guess i have to update these 5 places which is not associated with Progressbar at this moment. I guess this could cause some confuse at this moment, so hope to update this later in other issue. For example, in 5 places, 3 places uses until to 'electionEnd' and 2 places uses 'stageEnd' so we have to investigate this stuff before update this function.